webapp/src/main/resources/public/js/components/repositories/RepositoryRow.js

```diff
@@ -346,6 +356,28 @@ let RepositoryRow = React.createClass({
         return ui;
     },
 
+    /**
+     * @return {XML}
+     */
+    getCheckSLAOffLabel() {
```

Comment:
I think the label is a good idea for us but for other people that don't use the OOSLA feature it will make the UI overloaded and they'll wonder what that is about. 

Let's add an app config (off but default) to show it. see `withAppConfig`+ `ReactAppConfig`. This is a config you set at the server level. See `#l10n.security.oauth2.enabled=true` in `config/webapp/application.properties` and `OAuth2Config`

Reply:
adding server side app config for sla checker.